rust/cx-sdk/src/lib.rs

```diff
@@ -31,6 +34,19 @@ mod util;
 
 const ENV_CORALOGIX_REGION: &str = "CORALOGIX_REGION";
 
+const AUTHORIZATION_HEADER_NAME: &str = "authorization";
+
+pub(crate) const SDK_VERSION_HEADER_NAME: &str = "cx-sdk-version";
```

Comment:
those shouldn't be pub

Reply:
can these also not live where they are being used?

Reply:
Done